prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MS/ms-doh.yml

```diff
@@ -0,0 +1,108 @@
+# To submit this to localhost, run:
```

Comment:
same comment as ND and IL. Want to store this back in prime-router/settings/prod or prime-router/settings/staging as there are new features that platform has put in that auto checks the transforms